easybuild/easyconfigs/s/SCons/SCons-3.0.1-intel-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
+
+name = 'SCons'
+version = '3.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.scons.org/'
+description = "SCons is a software construction tool."
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = [PYPI_SOURCE]
```

Comment:
It is also available in sourceforge, and previous easyconfigs used that URL. I am not arguing about switching to pypi, I am just wondering the rationale behind

Reply:
I based this on an existing easyconfig which is already using `PYPI_SOURCE`:

```
$ grepi PYPI easybuild/easyconfigs/s/SCons/*eb
easybuild/easyconfigs/s/SCons/SCons-2.5.1-intel-2017a-Python-2.7.13.eb:12:source_urls = [PYPI_SOURCE]
```

Not sure which I prefer, but I guess consistently would be nice, yes.

I'll look into it.

Reply:
Updated to use SourceForge for consistency, checksums are not affected.